### PR TITLE
Cspell projects and lessons

### PR DESCRIPTION
**Pull request by nicktu12**
Utilizing cspell, I found spelling errors throughout the curriculum. This audit only includes spelling checks for markdown files in the /lessons and /projects directories - the archive folders of both of these directories has been omitted from this audit.

### CODE REVIEW

File: lessons/module-1/return-and-srp.md [hunk 1]
```diff
@@ -56,9 +56,9 @@ We can use this to our advantage when trying to stop the execution of a program
 
 These examples may be familiar, but they become more important when considering how to make our functions adhere to the single responsibility principle. 
 
-### Single Responsibilty Principle
+### Single Responsibility Principle
 
-The single responsibility principle is the first principle of object oriented design. The term was coined by Robert C. Martin, a computer programmer and co-author of the Agile Manifesto. It states that 
+The single eesponsibility principle is the first principle of object oriented design. The term was coined by Robert C. Martin, a computer programmer and co-author of the Agile Manifesto. It states that
```
Comment (niksseif) [inline lessons/module-1/return-and-srp.md]: @nicktu12 - Thanks for reviewing the lessons and fixing the typos, I noticed a small typo: 'eesponsibility' should be 'responsibility'.

**Review by niksseif — COMMENTED** [review]
Thank you for reviewing and fixing typos. I've gone through your changes for M1 and everything looks great. I did spot one small typo, which I've noted in a comment for you

File: lessons/module-1/return-and-srp.md (hunk 1 above)
Comment (niksseif) [reply lessons/module-1/return-and-srp.md]: It looks like you've already resolved this here [c04668e](https://github.com/turingschool/front-end-curriculum/pull/756/commits/c04668e6bd1e5fb50c662988d822e3787d2e8ff3) - Thank you 🙏🏼

**Review by Kalikoze — APPROVED** [review]
Just finished reviewing Mod 2 changes for lessons & projects and it looks all good to me.  Spelling is hard! 😅  Appreciate you for going through and doing all of this!

**Pull request merged**